Project.toml

```diff
@@ -17,15 +17,31 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
-StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
I'm not totally sure but it looks like uses of StaticArrays are split between "we also need Manifolds.jl" and "that's easily avoidable".

Reply:
if we can avoid it I would not mind, also if we only need it with manifolds would be great; I am not sure why we needed it.